api/models/PushNotification.js

```diff
@@ -73,5 +76,9 @@ module.exports = bookshelf.Model.extend({
 
   textForApprovedJoinRequest: function (community, actor) {
     return `${actor.get('name')} approved your request to join ${community.get('name')}`
+  },
+
+  setupDeviceIds: function () {
```

Comment:
empty function, is this supposed to be here?

Reply:
ah good catch